CHANGELOG-0.1.0.md

```diff
@@ -0,0 +1,8 @@
+# v0.1.0
+
+## Initial release Changelog
+
+* Add a external resize controller which monitors Persistent volume claims and performs
```

Comment:
Link to the PR # here.

I want to move the things that were under "Major changes" in previous release notes, to this section

Reply:
I have linked PRs. Since there aren't any previous release notes here - we can start with this list we have?